packages/pwa-buildpack/lib/queries/getAvailableStoresConfigData.graphql

```diff
@@ -0,0 +1,8 @@
+query availableStoresConfigData {
```

Comment:
Pointing out here that @tjwiebell is in the process of moving most/all of the queries next to their respective talons. Obviously this and other build time queries are unique and I don't know the plan for them. I assume they will just become `.gql` files next to the files that use them.